Source/AppAuthCore/OIDAuthorizationRequest.h

```diff
@@ -134,9 +134,14 @@ extern NSString *const OIDOAuthorizationRequestCodeChallengeMethodS256;
  */
 @property(nonatomic, readonly, nullable) NSDictionary<NSString *, NSString *> *additionalParameters;
 
+/*! @brief The client's additional authorization header.
+    @see https://tools.ietf.org/html/rfc6749#section-3.2
+ */
+@property(nonatomic, readonly, nullable) NSDictionary<NSString *, NSString *> *additionalHeaders;
```

Comment:
I don't think if we want to add `additionalHeaders` to `OIDAuthorizationRequest`. Typically, IdPs don't take additional headers for authorization requests because doing so could restrict the [external user agents](https://www.rfc-editor.org/rfc/rfc8252.txt). For example, AppAuth-iOS's [OIDExternalUserAgentIOS](https://github.com/openid/AppAuth-iOS/blob/master/Source/AppAuth/iOS/OIDExternalUserAgentIOS.m#L107) uses `ASWebAuthenticationSession`, which doesn't allow the developer to modify the request headers.

Let me know if you agree. If you do, let's revert all of the changes to `OIDAuthorizationRequest`.

Reply:
I agree. I reverted the changes in `OIDAuthorizationRequest`.